### PR TITLE
Aliens cannot be husked

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Xenomorphs now innately have the UNHUSKABLE trait.

## Why It's Good For The Game

Xenomorphs behave rather uniquely compared to humans, especially in their health department. Various stupid oddities in it result in the fact that xenomorphs almost always husk nigh-instantly after death - they have 300+ burn or whatnot, when a normal human has 200 burn at the point of death. This is annoying for some niche interactions, such as reviver implant, which works on xenos, never being able to do its job because it can't revive husks.

That's the walance bit. The other bit is that, uh, how do you husk an exoskeleton. I dunno seems odd.

TL;DR: It arises from buggy behavior, breaks niche interactions, and makes no 'realistic' sense anyways.

## Changelog

:cl:
code: Aliens cannot be husked
/:cl:

### CODE REVIEW

**Comment by Vect0r2** [comment]
How... How is this a code improvement?

**Comment by carlarctg** [comment]
> How... How is this a code improvement?

sometimes i dont know how to label prs

**Comment by AtomTheProphet** [comment]
This seems closer to a fix of an oversight to me, tbh.

**Review by MrMelbert — CHANGES_REQUESTED** [review]
I would rather the husk threshold be changed to be based off of health / total limb health than aliens just be made outright unhuskable

**Comment by MrMelbert** [comment]
(Aliens have unique husked sprites and seeing a bunch of toasted aliens when someone goes wild with a flamethrower is fun)

image

**Comment by carlarctg** [comment]
how ui toast exdoskeleotn thats lrp

**Comment by carlarctg** [comment]
this whole thing is fucking shitcode i have worse things to do with my time
